source/MaterialXRenderGlsl/TextureBaker.cpp

```diff
@@ -21,18 +21,8 @@ namespace {
 const string SRGB_TEXTURE = "srgb_texture";
 const string LIN_REC709 = "lin_rec709";
 const string BAKED_POSTFIX = "_baked";
-
-StringVec getRenderablePaths(ConstDocumentPtr doc)
-{
-    StringVec renderablePaths;
-    std::vector<TypedElementPtr> elems;
-    findRenderableElements(doc, elems);
-    for (TypedElementPtr elem : elems)
-    {
-        renderablePaths.push_back(elem->getNamePath());
-    }
-    return renderablePaths;
-}
+const string SHADER_PREFIX = "SR_";
+const string UDIM_PREFIX = "_";
```

Comment:
Here I'd recommend the name DEFAULT_UDIM_PREFIX for this constant, in order to clarify that it can be overridden by the user.